TF_Settings_Web/src/Components/Pages/Camera/Masking/handRendering.ts

```diff
@@ -0,0 +1,90 @@
+import { v4 as uuidgen } from 'uuid';
```

Comment:
Side note but might want to add this to the prettier import order regex so we can control where to put it (not really important though)